src/services/CertificateGenerationService.ts

```diff
@@ -192,19 +192,19 @@ class CertificateGenerationService {
 
     /**
      * Retrieves the odometer history for a given VIN from the Test Results microservice
-     * @param vin - VIN for which to retrieve odometer history
+     * @param systemNumber - systemNumber for which to retrieve odometer history
      */
-    public async getOdometerHistory(vin: string) {
+    public async getOdometerHistory(systemNumber: string) {
```

Comment:
No affected tests?

Reply:
there are only tests without make an model, thus..the getOdometerHistory function is stubed to return undefined everywhere